examples/src/examples/loaders/gsplat-many.example.mjs

```diff
@@ -71,7 +71,6 @@ assetListLoader.load(() => {
 
     // instantiate guitar with a custom shader
     const guitar = assets.guitar.resource.instantiate({
-        fragment: files['shader.frag'],
```

Comment:
I think it would be good to do the color animation in fragment shader as an example on how to do it .. it handles long splats better than per vertex.